classes/gettext.oeclass

```diff
@@ -5,9 +5,12 @@
 ##
 ## @useflag nls Enable nls
 
-CLASS_DEPENDS =+ "${DEPENDS_GETTEXT}"
-DEPENDS_GETTEXT = ""
-DEPENDS_GETTEXT:USE_nls = "gettext native:gettext-utils libiconv libintl"
+CLASS_DEPENDS =+ "${DEPENDS_GETTEXT_NLS} ${DEPENDS_GETTEXT_NATIVE}"
+DEPENDS_GETTEXT_NATIVE = ""
+DEPENDS_GETTEXT_NATIVE:native = "native:gettext-utils"
```

Comment:
So if you have USE_nls flag disabled, you need native:gettext-utils for native recipes, but nothing for other recipe types?  Why/how is that?